src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Tournaments/TournamentViewModel.cs

```diff
@@ -205,10 +205,17 @@ public bool IsDivisionsCountMax
         /// </summary>
         /// <param name="tournament"> Domain object </param>
         /// <returns> View model object </returns>
+        /// 
+
+        public string AlarmMessage { get { return !this.IsArchived ? "Tournament was successfully archived": "Tournament was successfully unarchived"; } set { } }
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use the 'value' parameter in this property set accessor declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3237)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS108)

Reply:
Pleaese delete line 209 and set accessor